README.md

```diff
@@ -69,4 +77,11 @@ These repos are marked as miscellaneous because they do not affect the developme
 - [repo-labels.json](repo-labels.json): A json file that can be used to add the specific GitHub issue labels code.gov uses on their projects
 - [repository_management.md](repository_management.md): This is a living document explaining how we collaboratively manage the Code.gov platform repositories
 
-## Thanks!
+
+## Questions?
+If you have questions, please feel free to contact us:  
+* Open an issue
```

Comment:
Please go ahead and link this to the new issue page for this repo. It looks like that was missed before.

Reply:
I copied this text from the repo template to update it and I forgot to add the link. Just added it!